src/olympia/api/tests/test_fields.py

```diff
@@ -150,22 +150,19 @@ def test_get_attribute_source(self):
         assert result == expected
 
     def test_get_attribute_empty_context(self):
-        mock_serializer = serializers.Serializer()
-        mock_serializer.context = {}
+        mock_serializer = serializers.Serializer(context={})
         field = self.field_class()
         self._test_expected_dict(field, mock_serializer)
 
     def test_field_get_attribute_request_POST(self):
         request = Request(self.factory.post('/'))
-        mock_serializer = serializers.Serializer()
-        mock_serializer.context = {'request': request}
+        mock_serializer = serializers.Serializer(context={'request': request})
```

Comment:
@diox can you explain these tests?  `mock_serializer` isn't used after being initialized, and if I take it out then both `test_field_get_attribute_request_POST` and `test_field_get_attribute_request_GET` are the same, with no request.

Reply:
(asking because looks like you added them)

Reply:
I think the tests are wrong. I think they should be doing this instead:
```
field = self.field_class()
field.context = {'request': request}
```

And not include the 2 `mock_serializer` lines at all.

Reply:
setting `field.context` doesn't work in 3.6.4 (see the initial test fails), but I get your meaning.

Reply:
Ah, yeah, if we can't set `context` directly then using the serializer like you did makes sense.